aikau/src/main/resources/alfresco/renderers/PublishAction.js

```diff
@@ -86,6 +87,29 @@ define(["dojo/_base/declare",
       publishTopic: "ALF_ITEM_SELECTED",
 
       /**
+       * Indicates that this should only be displayed when the item (note: NOT the renderer) is
+       * hovered over.
+       *
+       * @instance
+       * @type {boolean}
+       * @default
+       */
+      onlyShowOnHover: false,
+
+      /**
+       * Overrides the default to handle onlyShowOnHover for the current item.
+       *
+       * @instance
+       */
+      postCreate: function alfresco_renderers_PublishAction__postCreate() {
+         this.inherited(arguments);
+         if (this.onlyShowOnHover === true)
+         {
+            domClass.add(this.domNode, "hover-only");
```

Comment:
Should be using BEM for this, so the class would be `.alfresco-renderers-PublishAction--show-on-hover` or very similar.